robot/basestation/templates/Navbar.html

```diff
@@ -45,10 +45,25 @@
         <button type="button" id="send-serial-btn" class="btn btn-primary">Send serial cmd</button>
       </div> -->
 
-      <div class="col-md-7">
-        <div class="row" style="color:white">
-          <div class="col-md-3">BATTERY VOLTAGE: </div>
-          <div class="col-md-1" id="battery-voltage">-</div>
+  <div class="col-md-3">
+    <div class="row" style="color:white">
+      <div class="col-md-4">BATTERY VOLTAGE: </div>
+      <div class="col-md-2" id="battery-voltage">-</div>V
+    </div>
+  </div>
+
+  <div class="col-md-5">
+    <div class="row" style="color:white">
+      <div class="col-md-3">BATTERY TEMPS: </div>
+      <div class="col-md-5">
+        <div class="row">
+          T2:<div class="col-md-3" id="battery-temp-1">-</div>&#8451;
+          T1:<div class="col-md-3" id="battery-temp-2">-</div>&#8451;
```

Comment:
It says `T1` for `battery-temp-2` and `T2` for `battery-temp-1` but `T3` for `battery-temp-3`
Kind of confusing

Reply:
If you could find a way to clean up the html/css I would love you, I slapped this stuff together and am not proud of it